snooty/test_project.py

```diff
@@ -63,17 +63,19 @@ def test() -> None:
         # Ensure that the correct pages and assets exist
         index_id = FileId("index.txt")
         assert list(backend.pages.keys()) == [index_id]
+        # Confirm that no diagnostics were created
+        assert backend.diagnostics[index_id] == []
         code_length = 0
         checksums: List[str] = []
         index = backend.pages[index_id]
-        assert len(index.static_assets) == 2
+        assert len(index.static_assets) == 1
         assert not index.pending_tasks
         for node in ast_dive(index.ast):
             if isinstance(node, n.Code):
                 code_length += len(node.value)
             elif isinstance(node, n.Directive) and node.name == "figure":
                 checksums.append(node.options["checksum"])
-        assert code_length == 345
+        assert code_length == 400
```

Comment:
What's the reason for this change? Is it because of the accidental removal of line 0 that you fixed?

Reply:
The length of `index.static_assets` drops by one, because our literally included file (`test_project/source/driver-examples/DocumentationExamples.cs`) is [no longer a static_asset](https://github.com/mongodb/snooty-parser/blob/master/snooty/parser.py#L553). I don't believe that `code_length` increases because of that `start_after` OBOE, since the `index.txt` specifies the `start_after` text. I'll look into this.

Reply:
It was due to a misconfigured `dedent` conditional, and has been fixed!